scripts/lint.py

```diff
@@ -249,11 +249,6 @@ def lint_line(
             if not app_id.startswith("rerun_example_") and not app_id == "<your_app_name>":
                 return f"All examples should have an app_id starting with 'rerun_example_'. Found '{app_id}'"
 
-    # Methods that return Self should usually be marked #[inline] or #[inline(always)] since they indicate a builder.
-    if re.search(r"\(mut self.*-> Self", line):
-        if prev_line_stripped != "#[inline]" and prev_line_stripped != "#[inline(always)]":
-            return "Builder methods impls should be marked #[inline]"
```

Comment:
that wasn't a bad lint though, no? could just do `NOLINT` on `without_query_and_fragment` 🤷

Reply:
I added it back before rustc did auto-inlining of trivial functions. It _should_ not be needed anymore